src/onlyspans.polyleads.ui/src/components/DocumentsManager/DocumentsTable/DocumentsTable.tsx

```diff
@@ -0,0 +1,170 @@
+'use client';
+
+import useGet from '@/hooks/useGet';
+import ServiceSymbols from '@/data/constant/ServiceSymbols';
+import { IDocumentsTableVM } from '@/components/DocumentsManager/DocumentsTable/DocumentsTableVM';
+import React from 'react';
+import { observer } from 'mobx-react-lite';
+
+import {
+  ColumnFiltersState,
+  flexRender,
+  getCoreRowModel,
+  getFilteredRowModel,
+  getPaginationRowModel,
+  getSortedRowModel,
+  SortingState,
+  useReactTable,
+  VisibilityState,
+} from '@tanstack/react-table';
+
+import { Button } from '@/components/ui/button';
+import { Input } from '@/components/ui/input';
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from '@/components/ui/table';
+import { RowData } from '@tanstack/table-core';
+import DataTableViewOptions from '@/components/DocumentsManager/DocumentsTable/DataTableViewOptions/DataTableViewOptions';
```

Comment:
```suggestion
import DataTableViewOptions from './DataTableViewOptions/..';
```

на относительный путь в общем, IDocumentsTableVM тоже